tests/repo/reservedname.c

```diff
@@ -32,6 +35,9 @@ void test_repo_reservedname__includes_shortname_when_requested(void)
 	git_buf *reserved;
 	size_t reserved_len;
 
+	if (!cl_sandbox_supports_8dot3())
+		clar__skip();
```

Comment:
Actually, this really concerns me.  There's nothing here that should ever fail, ever never.  This test runs *on Unix*.